src/remote_action_middleware.js

```diff
@@ -1,6 +1,12 @@
 export default (socket) => (store) => (next) => (action) => {
```

Comment:
So this is only intercepting remote actions right? I think most / all of our client actions are remote, but should we be blocking all actions in general? I don't think this would catch following links, for example. Maybe better to create and insert a new middleware.